### ISSUE TITLE
Remote API v2 inconsistent URL pattern

### ISSUE THREAD

**Issue by kazukisan** [webanno/webanno#798]
Delete document requires a slash at the end while all other end points don't.

```
DELETE /api/v2/projects/{projectId}/documents/{documentId}/
```

Pull request coming.

**Comment by reckart** [comment]
Thanks! Btw. for these trivial changes, doing just a PR is fine. I can rewrite the commit comment and include the PR number when squash-merging the PR. For non-trivial PRs, having a separate issue is certainly preferable because there is a "risk" of having multiple commits in the PR and then rewriting the commit messages is rather tedious.

**Comment by kazukisan** [comment]
Thanks for the note !

**Comment by reckart** [comment]
Merged and cherry-picked into 3.3.x.
